### ISSUE TITLE
Object detection evaluation warning

### ISSUE THREAD

**Issue by modanesh** [tensorflow/tensorflow#12553]
Hi,
Here is my system details:

- macOS10
- no GPU
- python3.6
- tensorflow1.3
- installed tf from source

I'm running the evaluation of object detection, but every time I run into this warning:

```
WARNING:root:The following classes have no ground truth examples: 0
/Users/Mohamad/Projects/Python/tensorflow/models/object_detection/utils/metrics.py:145: RuntimeWarning: invalid value encountered in true_divide
  num_images_correctly_detected_per_class / num_gt_imgs_per_class)
```
I want to know how to solve this warning? Is it something serious?

Thanks.

**Comment by andydavis1** [comment]
@derekjchow can you comment on this one?

**Comment by Prasad9** [comment]
@modanesh , were you able to fix that problem?
I am having the same problem. I did the training of Oxford-IIIT pets dataset as mentioned in the dataset. Then during evaluation I am getting the same error. 
My system details:
Ubuntu 16.04, Python 3.5, Tensorflow 1.1, I have GPU support.

Also, after that warning comes, the evaluation stops for me. Please can somebody help me on this?

**Comment by modanesh** [comment]
@Prasad9 No, I haven't and I didn't find anything useful. I'm still waiting for someone to shade light on it.

**Comment by HungLingChen** [comment]
I have the same issue!!!

**Comment by derekhh** [comment]
See https://github.com/tensorflow/models/issues/1696 for an explanation. Basically this warning is expected as label map ids now start with 1.

**Comment by Prasad9** [comment]
@derekhh, this is how my label map id file looks like. 

```
item {
  id: 1
  name: 'One'
}

item {
  id: 2
  name: 'Two'
}

item {
  id: 3
  name: 'Three'
}

item {
  id: 4
  name: 'Four'
}

item {
  id: 5
  name: 'Five'
}

item {
  id: 6
  name: 'Six'
}

item {
  id: 7
  name: 'Seven'
}

item {
  id: 8
  name: 'Eight'
}

item {
  id: 9
  name: 'Nine'
}

item {
  id: 10
  name: 'Zero'
}

```

I am not having any id with zero. Also, when I evaluate, I get two .tfevents file and during the third one, this warning(may be error) comes and then there happens to be no more processing after that. I can check in tensorboard the evaluation done for the first two images.

INFO:tensorflow:Restoring parameters from /my/path/train/model.ckpt-200000
INFO:tensorflow:Restoring parameters from /my/path/train/model.ckpt-200000
WARNING:root:The following classes have no ground truth examples: 0
/my/path/models/object_detection/utils/metrics.py:145: RuntimeWarning: invalid value encountered in true_divide

**Comment by naimmd** [comment]
Have you  changed the label_id_offset from 0 to 1 ?
./object_detection/evaluator.py:187:     categories=categories, label_id_offset=0)

                                                    **categories=categories, label_id_offset=1)**

**Comment by RenukaNale** [comment]
same error when running eval.py 
WARNING:root:The following classes have no ground truth examples: 0
any solution even i try to run eval.py and train.py parallely still same error.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by bignamehyp** [comment]
@derekjchow can you comment on this one?

**Comment by bignamehyp** [comment]
Close due to inactivity.
